client/java/src/main/java/io/openlineage/client/circuitBreaker/ExecutorCircuitBreaker.java

```diff
@@ -17,9 +19,11 @@
 public abstract class ExecutorCircuitBreaker implements CircuitBreaker {
 
   private Integer circuitCheckIntervalInMillis;
+  private Optional<Duration> timeout;
 
-  public ExecutorCircuitBreaker(Integer circuitCheckIntervalInMillis) {
+  public ExecutorCircuitBreaker(Integer circuitCheckIntervalInMillis, Optional<Duration> timeout) {
```

Comment:
Additional constructor instead of passing optional seems to be the java pattern.